code/src/Components/ThoughtElements.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import Moment from 'moment';
+
+
+
+const ThoughtElement = ({ thought, handleLike }) => {
+  console.log(thought)
+
+
+  return (
+    <section className='thought-container grow shadow-20'>
```

Comment:
don't know really how to comment on the hover effect for the thoughts containers since it's not done in the css file but i think maybe that the scaling makes so that the like button is only clickable on the right side of the button. When the left side of the like button is pressed the container shinks and then grows again which is also quite abrupt.